utils/check_api_reference.py

```diff
@@ -133,7 +133,9 @@
 if (
     extra := set(series)
     .difference(expr)
-    .difference({"to_pandas", "to_numpy", "dtype", "name", "shape", "to_frame"})
+    .difference(
+        {"to_pandas", "to_numpy", "dtype", "name", "shape", "to_frame", "is_empty"}
+    )
```

Comment:
Would it make sense to get directly the set of methods that a polars Series has but are not an Expr? It would avoid having to manually add method names.

Something like:

```py
series_only = [i for i in dir(pl.Series) if not i[0].isupper() and i[0] != "_" and i not in dir(pl.Expr)]
```

Let me know what you think

Reply:
maybe, but there's also value in this being manual, so we can inspect whether it makes sense that a given method is missing